tools/sign_release.sh

```diff
@@ -87,8 +71,18 @@ curl -F "input=@$BUILD_DIR/app-amazonWebview-release-unsigned.apk" \
 zipalign -v 4 $BUILD_DIR/$FINAL_NAME $BUILD_DIR/app-amazonWebview-release-aligned.apk
 mv -f $BUILD_DIR/app-amazonWebview-release-aligned.apk $BUILD_DIR/$FINAL_NAME
 
+
+# We don't use `-Werr` (treat warnings as errors) because errors related to certain AndroidX
+# files are expected here.
+#
+# When the build is signed by Autograph, files inside the META-INF folder are not signed (due to
```

Comment:
Can we be specific about the outdated dependency? It's because they use jarsigner/apksigner v1 instead of v2, right?